### ISSUE TITLE
Feature Request - Will Pay

### ISSUE THREAD

**Issue by ph3l1x** [AHAAAAAAA/PokemonGo-Map#2922]
I am willing to pay someone that can create a version of this map that will allow for session capability that will allow concurrent users on a single database. Is there anyone that would be willing to do this?

**Comment by miaojs** [comment]
What you're describing already works. I don't think you meant what you wrote. I think you meant on a single instance.

Currently you can run multiple instances each with their own session on a different port, all running on the same database with results showing across all instances.

**Comment by ph3l1x** [comment]
Yes, on a single instance. There are about 20 people I have in a group. And would like them all to be able to visit this page and basically have this work for them by going to page and just setting a marker down.

**Comment by Paralysing** [comment]
There was a pr that was closed a little bit ago, but it has whay you descrive, check #2150

**Comment by ph3l1x** [comment]
That is a bit closer. But basically looking for something that functioned just like pokevision did. Anyone can go to page click spot on map and get a list of items nearby. Is that possible to do?

**Comment by SethG911** [comment]
Yes it is possible just by running separate instances on different ports. Then use URL remapping to assign each port instance to its own URL (likely a subdomain) for each user. If all users are sharing a single DB they'll be able to see other users' searches as well but still control their instanced location.

**Comment by ph3l1x** [comment]
I think that something like that would work quite well. I would be willing to pay you if this is something you could maybe do for me?

**Comment by SethG911** [comment]
No, sorry. It would be best for you to work your way through it anyway. That way you can maintain it and not have to rely on someone else to keep it running for you.

**Comment by coolaj86** [comment]
I've actually done most of that here https://github.com/Daplie/node-pokemap (allows multiple users simultaneously) and would be willing to add the functionality to allow the group of users to share data via WebRTC or GunDB or something like that.
